pkg/apis/pipeline/v1beta1/pipelineref_types.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+// PipelineRef can be used to refer to a specific instance of a Pipeline.
+// Copied from CrossVersionObjectReference: https://github.com/kubernetes/kubernetes/blob/169df7434155cbbc22f1532cba8e0a9588e29ad8/pkg/apis/autoscaling/types.go#L64
```

Comment:
I am not sure what this is referring to 🤦‍♀️ What is copied from CrossVersionObjectReference?

Reply:
oh, its just a copy/paste from previous location 🙃 

not blocking this PR, but unless it serves any helpful details, we can drop it

Reply:
removed!